lib/asset-rev.js

```diff
@@ -24,9 +25,19 @@ function AssetRev(inputTree, options) {
   this.ignore = options.ignore;
   this.description = options.description;
 
+  // first pass - excludes replaceable source code
+  this.exclude = exclude.slice();
+  this.replaceExtensions.forEach(function(ext) { this.exclude.push('**/*.' + ext); }, this);
```

Comment:
Is `this.exclude` a list of glob matchers?  If it is, we should anchor these strings we are pushing to the end of the filename (I wouldn't want to match `assets/.json-generator/something.png` or whatever).